README.md

```diff
@@ -91,7 +91,7 @@ Unlike `protolude`, we are:
 1. Not trying to be as general as possible (thus we don't export much from
    [`GHC.Generics`](https://github.com/sdiehl/protolude/blob/41710698eedc66fb0bfc5623d3c3a672421fbab5/src/Protolude.hs#L365)).
 2. Not trying to maintain every version of `ghc` compiler (only the
-   [latest 3](https://github.com/serokell/universum/blob/b6353285859e9ed3544bddbf55d70237330ad64a/.travis.yml#L15)
+   [latest 4](https://github.com/serokell/universum/blob/b6353285859e9ed3544bddbf55d70237330ad64a/.travis.yml#L15)
```

Comment:
I wouldn't say it's exactly "latest 4" what we support...

Reply:
Well, currently we do support exactly latest four and this comment was wrong, so I fixed it.
Do you want it to be rephrased to something like "only the most recent ones", so that we don't have to update it every time the list of supported versions changes?

Reply:
"at least the latest 3"

Reply:
Ok, updated.